### ISSUE TITLE
makewindows --> illegal number bug

### ISSUE THREAD

**Issue by leone93** [arq5x/bedtools2#1017]
Hi, 
from bedtools version 2.29 on is not possible to use makewindows with the option 
`bedtools makewindows -g hg19.txt -w 1000000`
It will always report a bug 

> ". Exiting...illegal number "_(whathever the number)_

Version until 2.28 works great.

**Comment by arq5x** [comment]
Please use the latest version.

**Comment by arq5x** [comment]
bin/bedtools --version
    bedtools v2.30.0

    head genomes/human.hg19.genome
    chr1	249250621
    chr2	243199373
    chr3	198022430
    chr4	191154276
    chr5	180915260
    chr6	171115067
    chr7	159138663
    chrX	155270560
    chr8	146364022
    chr9	141213431

    bin/bedtools makewindows -g genomes/human.hg19.genome -w 1000000 | head
    chr1	0	1000000
    chr1	1000000	2000000
    chr1	2000000	3000000
    chr1	3000000	4000000
    chr1	4000000	5000000
    chr1	5000000	6000000
    chr1	6000000	7000000
    chr1	7000000	8000000
    chr1	8000000	9000000
    chr1	9000000	10000000

**Comment by leone93** [comment]
Guys, I understand the problem.
I made the file in windows and the newline character 0xD is not recognized from the new version of bedtools.
Instead the old version of bedtools (until 2.28) and all the other linux tools has no problem with it.
[chr_list.txt](https://github.com/arq5x/bedtools2/files/9867502/chr_list.txt)
`cat -te chr_list.txt`
```
CHR01_Om^I94519681^M$
CHR02_Om^I88319851^M$
CHR03_Om^I82634242^M$
CHR04_Om^I55838738^M$
CHR05_Om^I67115614^M$
CHR06_Om^I66880125^M$
CHR07_Om^I62504570^M$
CHR08_Om^I60554199^M$
CHR09_Om^I44233928^M$
CHR10_Om^I50475751^M$
CHR11_Om^I63596752^M$
CHR12_Om^I52345910^M$
```

**Comment by jmarshall** [comment]
Not accepting CR-LF newlines in genome files is the same problem as #998 and has been fixed on **master** by PR #999 (but that fix has not yet appeared in a release).
